client/components/SocialClub/SocialClubPage.js

```diff
@@ -4,38 +4,33 @@ import toastr from 'toastr';
 
 // actions
 import { getClub, joinClub } from '../../actions/socialClubActions';
-import { sendEvent } from '../../actions/createEventActions';
+import { createEvent } from '../../actions/eventActions';
 
 // components
 import PageHeader from './PageHeader';
 import EventList from '../Events/EventList';
 
 
-
 const attendees = [
```

Comment:
Are we still making use of these hard-coded variables? Thought we have left them lol.

Reply:
I only left them so we can remember to get the slack usernames of attendees to an event into our state. That has not been done yet.